core/tests/protractor_utils/LearnerDashboardPage.js

```diff
@@ -51,11 +52,12 @@ var LearnerDashboardPage = function() {
     element.all(by.css('.protractor-test-feedback-message'));
 
   this.get = function() {
-    return browser.get(LEARNER_DASHBOARD_URL);
+    browser.get(LEARNER_DASHBOARD_URL);
+    return general.waitForLoadingMessage();
```

Comment:
Wait for page to completely load after browser.get() calls

Reply:
You might want to make this an inline comment.

Reply:
Not sure you did this? (Also not sure if necessary...your call.)

If necessary, then ... follow-up PR :)

Reply:
I placed the comments in waitForLoadingMessage!